src/legacy-routes.js

```diff
@@ -176,19 +254,9 @@ const bulkAddressSummary = (dbApi: any, { logger, apiConfig }: ServerConfig) =>
   if (addresses.some((addr) => !isValidAddress(addr))) {
     return { Left: invalidAddress }
   }
-  const txList = await dbApi.bulkAddressSummary(addresses)
-  const transactions = txList.rows
-
-  const right = {
-    caAddresses: addresses,
-    caTxNum: transactions.length,
-    caBalance: {
-      getCoin: `${combinedBalance(transactions, addresses)}`,
-    },
-    caTxList: transactions.map(txToAddressInfo),
-  }
+  const right = await getAddressSummaryForAddresses(dbApi, addresses)
```

Comment:
if this isn't passed as-is to the result into the "Right" key, I would rename the variable as this is kind of misleading